modules/search/class.jetpack-search.php

```diff
@@ -142,17 +143,21 @@ public function search( array $es_args ) {
 		$end_time = microtime( true );
 
 		if ( is_wp_error( $request ) ) {
+			error_log(print_r($request,1));
 			return $request;
 		}
 
 		$response_code = wp_remote_retrieve_response_code( $request );
 
 		if ( ! $response_code || $response_code < 200 || $response_code >= 300 ) {
+			error_log("invalid API response");
 			return new WP_Error( 'invalid_search_api_response', 'Invalid response from API - ' . $response_code );
 		}
 
 		$response = json_decode( wp_remote_retrieve_body( $request ), true );
 
+		error_log(print_r($response,1));
```

Comment:
Let's not forget to remove these error log calls.

Reply:
Fixed!